src/templates/items/asi.hbs

```diff
@@ -33,18 +33,27 @@
         {{!-- Details Tab --}}
         <div class="tab details" data-group="primary" data-tab="details">
 
-            <h4 class="section-header first">{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.Header"}}</h4>
-
-            <blockquote class="quote">{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.InstructionRules"}}</blockquote>
-            <p>{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.Instruction"}}</p>
-            <div class="form-group stacked weapon-properties">
-                <label>{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.Label"}}</label>
-                {{#each config.abilities as |ability name|}}
-                <label class="checkbox" data-tippy-content="{{lookup (lookup ../config.abilities name) "tooltip"}}">
-                    <input type="checkbox" name="system.abilities.{{name}}" {{checked (lookup ../itemData.abilities name)}} /> {{ability}}
-                </label>
-                {{/each}}
+            <div class="bubble">
+                <h3 class="bubble-header">{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.Header"}}</h3>
+
+                <div class="bubble-info">
+                    <blockquote class="quote">{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.InstructionRules"}}</blockquote>
+
+                    <br />
+                    <div class="bubble">
+                        <h3 class="bubble-header">{{localize "SFRPG.ItemSheet.AbilityScoreIncrease.Instruction"}}</h3>
+
+                        <div class="form-group stacked bubble-info">
+                            {{#each config.abilities as |ability name|}}
+                            <label class="checkbox" data-tippy-content="{{lookup (lookup ../config.abilities name) "tooltip"}}">
```

Comment:
This toooltip lookup seems to be broken

Reply:
Tbh I'm not sure a tooltip is needed here. It's just selecting the appropriate ability, pretty self-explanatory